src/boot/init.ts

```diff
@@ -15,10 +16,10 @@ declare module "@vue/runtime-core" {
 }
 
 export default boot(async ({ app }) => {
-  const initActions = [
-    globalState.init()
-  ]
-  await Promise.all(initActions)
+  await appConfig.init()
+  const { nodeName } = (await appConfig.read());
+  globalState.setNodeName(nodeName);
+  await globalState.loadLangData()
```

Comment:
I don't get this refactoring, old and new code seems to be unrelated and changes do not correspond to commit messages :shrug:

Reply:
this part is related to the prev commit, where I have changed `Global` class